dataactcore/scripts/load_specific_fpds.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import datetime
+import logging
+import numpy as np
+import pandas as pd
+
+from dataactbroker.helpers.uri_helper import RetrieveFileFromUri
+
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.logging import configure_logging
+from dataactcore.scripts.pull_fpds_data import create_lookups, get_data
+
+from dataactvalidator.health_check import create_app
+
+logger = logging.getLogger(__name__)
+
+
+def get_param_cols(columns):
+    """ Get the columns that were provided in the file and return that list so we don't try to query non-existent cols
+
+        Args:
+            columns: The columns in the header of the provided file
+
+        Returns:
+            A dict containing all the FPDS query columns that the provided file has
+    """
+    possible_cols = {'agency_id': 'AGENCY_CODE:"{}" ', 'referenced_idv_agency_iden': 'REF_IDV_AGENCY_ID:"{}" ',
```

Comment:
Totally nitpicky and can be disregarded. To reduce the extra `"{}"` in each of the values, would it make more sense to have `possible_cols` just store the columns like `{'agency_id': 'AGENCY_CODE', ...}` and have `create_param_string` handle the string formatting `'{}:"{}" '`? Again, very trivial.